### PR TITLE
Render unsupported shape types in read-only mode.

### PR DESCRIPTION
**Pull request by trungleduc**
Screenshot 2022-11-07 121855

### CODE REVIEW

**Comment by martinRenou** [comment]
The RobotExample is the one I tried this morning, I'm surprised the robot arm is not visible on your screenshot? 

Screenshot from 2022-11-07 15-21-24

**Comment by trungleduc** [comment]
The robot is embedded into the `FCStd` file by using the `PropertyFileIncluded` property. We do not support this property yet, and also the `wrl` file format.

```
                <Property name="RobotVrmlFile" type="App::PropertyFileIncluded">
                    <FileIncluded file="kr500_1.wrl"/>
                </Property>
```

File: jupytercad/freecad/props/property_link.py [hunk 1]
```diff
@@ -14,4 +14,7 @@ def fc_to_jcad(prop_value: Any, jcad_file=None, fc_file=None) -> Any:
 
     @staticmethod
     def jcad_to_fc(prop_value: str, jcad_file=None, fc_file=None) -> Any:
-        return fc_file.getObject(prop_value)
+        try:
+            return fc_file.getObject(prop_value)
+        except Exception:
+            return None
```
[the remark this answers is not in the document]
Comment (trungleduc) [reply jupytercad/freecad/props/property_link.py:20]: Indeed!

**Review by martinRenou — APPROVED** [review]
Great!

**Pull request merged**